flink-python/lib/LICENSE.py4j

```diff
@@ -1,26 +0,0 @@
-Copyright (c) 2009-2018, Barthelemy Dagenais and individual contributors. All
```

Comment:
Are you sure we can delete this file? `flink-python/lib` contains a py4j-...zip file. And this zip file does not contain any license information.

It seems that these files end up in the `opt/python` directory of the Flink distribution. If we delete the files from here, might be gone from the dist?!

Reply:
As pointed by Chesney, these licenses also appear in the directory of licenses/ and so we could remove them. What do you think?

Reply:
But I think deleting them is not the right approach, as these files are needed when assembling the final binary distribution.
I would rather add a filter to the maven shade plugin to filter them out of the flink-python.jar.

Reply:
They also appear in the directory https://github.com/apache/flink/tree/master/licenses and so the binary distribution should include them.

Reply:
I think it should be fine to remove them. The copies at `flink-python/src/main/resources/META-INF/licenses` ensure they are contained in the flink-python jar and the distribution (and thus transitively the python distribution).

Reply:
Okay, thanks for the patient explanations. I wasn't aware that the files are located in the `/licenses` folder.